src/SprykerEco/Yves/FactFinderWebComponents/Theme/default/components/atoms/campaign/campaign.twig

```diff
@@ -0,0 +1,25 @@
+{% extends model('component') %}
+
+{% define config = {
+    name: 'campaign',
+    tag: 'campaign'
+} %}
+
+{% define data = {
+    properties: {
+        component: '',
+        answer: '',
+        question: '',
+        feedbacktext: ''
+    }
+} %}
+
+{% block body %}
+    <div class="box">
```

Comment:
I don't think we should wrap every component with box. Please remove.

Reply:
Removed